src/app.bs

```diff
@@ -16,7 +16,7 @@ import ./routes/auth
 
 app = express()
 
-app.set('views', path.join(__dirname, 'views'))
+app.set('views', path.join(__dirname, '..', 'src', 'views'))
```

Comment:
Maybe use `app.set('views', path.resolve('src', 'views'))`? Think it should work.